src/frollR.c

```diff
@@ -226,8 +226,9 @@ SEXP frollfunR(SEXP fun, SEXP obj, SEXP k, SEXP fill, SEXP algo, SEXP align, SEX
         REprintf("%s: %d:\n%s", __func__, i*nk+j+1, dans[i*nk+j].message[1]); // # nocov because no messages yet // change REprintf according to comments in #3483 when ready
       if (dans[i*nk+j].message[2][0] != '\0')
         warning("%s: %d:\n%s", __func__, i*nk+j+1, dans[i*nk+j].message[2]);
-      if (dans[i*nk+j].status == 3)
+      if (dans[i*nk+j].status == 3) {
```

Comment:
@jangorecki is this branch superfluous now that memory failure handling should be done by `R_alloc`?

Reply:
No, there are still errors can catch